cmd/apps/tekton_app.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) arkade author(s) 2020. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package apps
+
+import (
+	"fmt"
+
+	"github.com/alexellis/arkade/pkg"
+
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallTekton() *cobra.Command {
+	var tekton = &cobra.Command{
+		Use:          "tekton",
+		Short:        "Install Tekton pipelines and dashboard",
+		Long:         `Install Tekton pipelines and dashboard`,
+		Example:      `  arkade install tekton`,
+		SilenceUsage: true,
+	}
+
+	tekton.RunE = func(command *cobra.Command, args []string) error {
+		kubeConfigPath := getDefaultKubeconfig()
+
+		if command.Flags().Changed("kubeconfig") {
+			kubeConfigPath, _ = command.Flags().GetString("kubeconfig")
+		}
+
+		fmt.Printf("Using kubeconfig: %s\n", kubeConfigPath)
+
+		arch := getNodeArchitecture()
+		fmt.Printf("Node architecture: %q\n", arch)
```

Comment:
We need you to block installation on ARM architectures. See the approach of the Istio app

Reply:
No problem. I've done an initial scan of the [Tekton docs](https://github.com/tektoncd/pipeline/blob/master/docs/install.md) and haven't seen any architecture requirements. I'll double check.

Reply:
I've blocked and will update the PR. However, I used the string "amd64" rather than re-declaring the global variable IntelArch due to the previous declaration appearing in "istio_app.go".

Could this be opportunity to initialize the IntelArch variable in a different file? I'm unsure that a foo_app.go file should depend on variables from other bar_app.go files.

Reply:
>I've blocked and will update the PR. However, I used the string "amd64" rather than re-declaring the global variable IntelArch due to the previous declaration appearing in "istio_app.go".

That's now how package-scoped variables work in Go. There is no need to redeclare it, you can simply access it here, so please can you do that and remove "amd64" hard-coded a second time?